Purchasing.Mvc/Controllers/AjaxController.cs

```diff
@@ -1,21 +1,19 @@
 ﻿using System;
 using System.Linq;
-using System.Web.Mvc;
-using System.Web.SessionState;
 using Purchasing.Core.Services;
 using Purchasing.Mvc.Utility;
 using Purchasing.Mvc.Services;
 using UCDArch.Core.PersistanceSupport;
 using UCDArch.Core.Utils;
 using UCDArch.Web.ActionResults;
 using IdAndName = Purchasing.Core.Services.IdAndName;
+using Microsoft.AspNetCore.Mvc;
 
 namespace Purchasing.Mvc.Controllers
 {
     /// <summary>
     /// Controller for AJAX service calls
     /// </summary>
-    [SessionState(SessionStateBehavior.Disabled)] //Disabled so requests can run in parallel
```

Comment:
Searching for a building works, but I don't know if these changes will break something else.